control_panel_api/filters.py

```diff
@@ -86,8 +86,6 @@ def filter_queryset(self, request, queryset, view):
         if is_superuser(request.user):
             return queryset
 
-        accessible_buckets = S3Bucket.objects.filter(
-            users3buckets__user=request.user,
-        )
+        accessible_buckets = S3Bucket.objects.accessible_by(request.user)
 
-        return queryset.filter(s3bucket=accessible_buckets)
+        return queryset.filter(s3bucket__in=accessible_buckets)
```

Comment:
Good stuff on fixing this and the tests, sorry I thought Django ORM would behave similarly to Rails' one by converting this in an SQL `IN` statement. Thanks @jmoz

Reply:
👍